src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -285,57 +269,44 @@ trait EntityComponent {
       allEntitiesAction.flatMap(cloneEntities(destWorkspaceContext, _))
     }
 
-    private def insertEntitiesSql(entities: Seq[EntityRecord]): driver.api.DBIOAction[Unit, driver.api.NoStream, driver.api.Effect] = {
-      val insertBatches = entities.map { entity =>
-        sql"(${entity.id}, ${entity.name}, ${entity.entityType}, ${entity.workspaceId})"
-      }.grouped(batchSize).toSeq
-
-      DBIO.seq(insertBatches.map { insertBatch =>
-        val prefix = sql"insert into ENTITY (id, name, entity_type, workspace_id) values "
-        val suffix = insertBatch.reduce { (a, b) =>
-          concatSqlActionsWithDelim(a, b, sql", ")
-        }
-        concatSqlActions(prefix, suffix).as[Int]
-      }:_*)
+    def batchInsertEntities(workspaceContext: SlickWorkspaceContext, entities: Seq[EntityRecord]): ReadWriteAction[Seq[EntityRecord]] = {
```

Comment:
The new batchInsertEntities combined with the new schema allows us to insert batches of entities, select back their IDs (batching the selects as well), and then use those ids to insert attributes into the ENTITY_ATTRIBUTE table. We no longer need the attribute ids back, which was about 75% of the battle.